### DISCUSSION TITLE
RGB colour accuracy issue

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#3504] [General]
Hi everyone,

I have noticed when allowing RGB values to control the model colour the colour seems too light. Are we able to edit the RGB colour intensity? 

App Link:  https://inplacexr.tech/DOMUSLIVING%20/RGBTESTING%20/RGB%20testing%20fr.html 

I have attached an image demonstrating the colour difference. 
 
![RGB EXAMPLE IMAGE](https://user-images.githubusercontent.com/90825510/171161625-68ac494e-12b0-4f58-9d1c-83f8bc7386f6.JPG)
y

**Comment by elalish** [comment]
The material baseColor is not equivalent to rendered color (if it were, we could not shade the object to indicate its 3D shape). It merely represents the fraction of each color of light that is reflected, though there are also achromatic reflections and such based on Fresnel. The color/intensity of the lights as well as the camera exposure make as much difference to the output color as the baseColor does, just as in photography.
